### ISSUE TITLE
how have you found working with multiple tables in your database?

### ISSUE THREAD

**Issue by njons** [sunuwars/DJES#38]
We were discouraged from doing this by the mentors (and I do appreciate that storing only one value in an entire table may not be best practice), did you find it difficult to manage with multiple tables? Do you think it complicated any part of your code?

**Comment by sima-qian** [comment]
I found it surprisingly okay -- I think for what we wanted to achieve it would have been more difficult to squash everything into one table.

**Comment by sunuwars** [comment]
I agree with Eve, having 3 simple tables allowed us to practice using more sql statements ...
